scripts/react-compiler-compliance-check.ts

```diff
@@ -94,36 +131,25 @@ function runCompilerHealthcheck(src?: string): CompilerResults {
         cwd: process.cwd(),
     });
 
-    // Parse and then normalize via Set/Map to ensure true uniqueness
-    const parsed = parseCombinedOutput(output);
+    const results = parseHealthcheckOutput(output);
```

Comment:
Let's remove this redundant local var:


```suggestion
    return parseHealthcheckOutput(output);
```